frontend/rpm/template.go

```diff
@@ -537,3 +519,11 @@ func WriteSpec(spec *dalec.Spec, target string, w io.Writer) error {
 	}
 	return nil
 }
+
+func resolveFileName(path string, cfg dalec.ArtifactConfig) string {
```

Comment:
in the systemd PR I actually added a `ResolveName` method to `ArtifactConfig` that does exactly this, maybe you could use that instead?

Reply:
Thanks for pointing that out!